Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -104,11 +104,21 @@ extension PayPalViewModel: PayPalNativeCheckoutDelegate {
 
 extension PayPalViewModel: PayPalNativeShippingDelegate {
     
-    func paypal(_ payPalClient: PayPalNativeCheckoutClient, didShippingAddressChange shippingAddress: PayPalNativeShippingAddress) {
+    func paypal(
+        _ payPalClient: PayPalNativeCheckoutClient,
+        shippingActions: PayPalNativeShippingActions,
+        didShippingAddressChange shippingAddress: PayPalNativeShippingAddress
+    ) {
```

Comment:
```suggestion
    func paypal(
        _ payPalClient: PayPalNativeCheckoutClient,
        didChangeShippingAddress shippingAddress: PayPalNativeShippingAddress,
        withActions shippingActions: PayPalNativeShippingActions
    ) {
```
🧶 : Whenever we can, we should name delegate methods to read like a sentence i.e. `paypal:didChangeShippingAddress:withActions`.